libs/shared/ui-components/src/SocialCard/SocialCard.tsx

```diff
@@ -59,37 +68,41 @@ export const SocialCard: FC<TSocialCardProps> = ({
         return '';
     }
   };
+
   return (
     <Head>
       {/* twitter */}
       <meta
         name="twitter:card"
-        content={summaryLargeImage ? 'summary_large_image' : 'summary'}
+        content={
+          isCustom
+            ? getTwitterCardTypeName(props.twitterLargeImage)
+            : getTwitterCardTypeName()
```

Comment:
Could this be rewritten as

```suggestion
          twitterLargeImage 
            ? TWITTER_LARGE_CARD_NAME 
            : TWITTER_SMALL_CARD_NAME;
```

Reply:
If we pull `twitterLargeImage` over to the base type as you suggest, I would think so.